code/datums/status_effects/buffs.dm

```diff
@@ -414,6 +414,8 @@
 
 /datum/status_effect/regenerative_core/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IGNOREDAMAGESLOWDOWN, id)
+	to_chat(owner, span_warning("The tendrils of the regenerative core sink into your flesh, leaving dark markings where they dive."))
+	owner.adjustCloneLoss(10)
```

Comment:
this is pretty much unhealable. not a fan

Reply:
5 clone seems more reasonable but also ground puce crystals heal cloneloss + ghettoing cloneoxadone isnt that hard. burn medkit has rhigoxane which can activate it easily.

Reply:
Player knowledge of the ways we do have to heal clone damage is just low because clone damage is normally a complete non-factor. if it were to become relevant, they’d pick up wuickly enough.